src/CoreFx.Private.TestUtilities/src/System/AdminHelpers.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using Xunit;
+
+namespace System
+{
+    public static class AdminHelpers
+    {
+        /// <summary>
+        /// Runs the given command as sudo
+        /// </summary>
+        /// <param name="commandLine">The command line to run as sudo</param>
+        /// <returns> Returns the process exit code (0 typically means it is successful)</returns>
+        public static int RunAsSudo(string commandLine)
```

Comment:
I don't want to change the name of this as it doesn't match Unix terminology. When a Unix-centric developer is writing tests for Unix they're going to search for "sudo" in the code, not RunElevated. Even I started by looking for "sudo" when the original issue came up.

Additionally it makes reading the consuming code clearer- you don't have to know what RunAsSudo() does in a Unix test- Run() requires looking to be sure.

I'm also not sure we'll have a 1-1 analogy here for Windows. I'd rather introduce a "generically" named method when we actually write a cross platform test that always needs elevation- if such a thing even exists.